guide/payments/transactions.md

```diff
@@ -0,0 +1,88 @@
+
+#### 1. Get recipient address
+You need a valid address to send bitcoin. This can be shared by the recipient as a QR code, in plain text or as a payment link.
+
+#### 2. Creation
+The wallet application guides the user through collecting the required information (address and amount), and any optional information (what coins to send, fee options).
+
+#### 3. Signing
+The transaction needs to be signed by the [private key(s)]({{ '/guide/glossary/#private-key' | relative_url }}) of the input address(es) to be valid. The signing is often done in the same application and at the same time as the creation, but this does not have to be the case.
+
+#### 4. Broadcasting
+The transaction is broadcasted to a Bitcoin node, normally the one the wallet is connected to.
+
+#### 5. Validation
+The receiving node checks that the transaction is valid. In practice this means confirming that it was signed by the private key of the relevant address.
+
+#### 6. Propagation
+Once validated, the node passes the transaction on to other nodes in the network - returning a success message to the originator. The transaction is now in the *memory pool*, and remains there until confirmed.
+
+#### 7. Confirmations
+At some point the transaction is included in a block that is added to the blockchain. This counts as a first confirmation. The recipient can be confident that they will receive the bitcoin, and may even receive a pre-emptive notification from their wallet. How long the first confirmation takes is dependent on many factors, principally - how many other transactions are waiting to be confirmed and how big a fee the transaction will pay. A new block is confirmed roughly every ten minutes.
+
+Each additional block that is accepted by the network counts as an extra confirmation, making the transaction more secure by making it harder to undo. When a transaction has 6 confirmations, it is considered final.
+
+---
+
+## Transaction structure
+
+The most common Bitcoin transactions contain instructions to move funds from one address to another. Other types of transactions include creating multi-sig wallets or recording data on the blockchain.
+
+Once created, a transaction needs to be signed with the private key(s) of the sender(s), submitted to a node on the network for validation and propagation to other nodes. It then waits to be verified and included in a newly mined block, after which point it will be recorded on the blockchain. For this service, the sender will pay a fee for every transaction which goes to the node who mined the block as a reward.
+
+Technically, the transaction is data that contains the necessary information to execute on the blockcain. The principal properties are:
+
+- **Amount** -- the amount of bitcoin, in satoshis
+- **Inputs** -- the source(s)
+- **Outputs** -- the destination(s)
+- **Version** -- the code version this transaction follows
+- **Lock time** -- time until the transaction should be executed, optional
+
+### Input and outputs
+
```

Comment:
Should we have an illustration here? Usually, Inputs and Outputs are better understood visually, in my experience at least.